tests/components/vizio/test_config_flow.py

```diff
@@ -413,6 +462,39 @@
     assert result["type"] == data_entry_flow.RESULT_TYPE_ABORT
     assert result["reason"] == "updated_entry"
     assert hass.config_entries.async_get_entry(entry_id).data[CONF_NAME] == NAME2
+    assert hass.config_entries.async_get_entry(entry_id).data[CONF_APPS] == {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5134240)